rotkehlchen/tests/unit/decoders/test_uniswapv2.py

```diff
@@ -747,3 +748,196 @@ def test_uniswap_v2_remove_liquidity(database, ethereum_inquirer, eth_transactio
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[ADDY_4]])
+def test_uniswap_v2_swap_events_order(
+        database,
+        ethereum_inquirer,
+        eth_transactions,
+        ethereum_accounts,
+):
+    """Check that the order of swap events are consecutive."""
```

Comment:
Please add a description of the test case because we will read it again and won't know why this test it here. Mention that it handles the case of an approval in between of the trade